pkg/commands/product/botmanagement/common.go

```diff
@@ -0,0 +1,23 @@
+package botmanagement
```

Comment:
I think this package and each of the other new enable/disable/status packages under `pkg/commands/product` should be reduced to fewer files. When they each contained full implementations, separate files made sense. Now that the generic core does most of the work, I'd prefer to see something like `commands.go` and `commands_test.go` until a product gains unique functionality.

Reply:
Very good point... I'll do that now, and also use import aliases to remove unnecessary differences between the source files.